lib/env/config.js

```diff
@@ -60,6 +62,13 @@ module.exports = {
       return customConfigContent;
     }
     const configPath = getConfigPath();
-    return Promise.resolve(require(configPath)); // eslint-disable-line
+    try {
+      return Promise.resolve(moduleLoader.require(configPath));
+    } catch (e) {
+      if (e.code === 'ERR_REQUIRE_ESM') {
+        return (await moduleLoader.import(url.pathToFileURL(configPath))).default;
```

Comment:
This is more readable to me, it avoids return (await ....).default syntaxe which I think is less readable.
 
```suggestion
        return moduleLoader.import(url.pathToFileURL(configPath))).then(({ default }) => default);
```